.vib/wordpress/cypress/cypress/integration/wordpress_spec.js

```diff
@@ -96,17 +103,18 @@ it('checks the value of auto update status', () => {
 it('allows the upload of a file',() => {
   cy.login();
   cy.visit("wp-admin/upload.php");
-  cy.get("[role='button']").contains('Add New').click();
-  cy.get("[aria-labelledby]").contains('Select Files').should('be.visible');
+  cy.contains("[role='button']",'Add New').should('be.visible').click();
+  cy.contains("[aria-labelledby]",'Select Files').should('be.visible');
   cy.get('input[type=file]').selectFile('cypress/fixtures/images/test_image.jpeg',
     {force:true});
   cy.get('.attachment').should('be.visible');
 })
 
 it('allows to log out', () => {
   cy.login();
-  cy.get("#wp-admin-bar-logout .ab-item").click({
+  cy.get("#wp-admin-bar-logout .ab-item").should('exist').click({
```

Comment:
Same thoughts here about `exists`. I personally prefer `be.visible` over `exist` in most cases.

Reply:
See comment above. I'll try to see a better way.